database/seeders/TranslationsSeeder.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Seeder;
+
+class TranslationsSeeder extends Seeder
+{
+    public function run()
+    {
+        // TODO: I'm unsure what to do about this one, since the translation
+        // tables won't exist by the time this seeder is running.
```

Comment:
If the seeders are meant to represent the state of the database structure today—and not track with its changes over time—I think you can omit this altogether. We've removed translation functionality from the Visit page. And when we remove functionality from the application we remove all references to it in code and the database so we don't have stale code sitting around that we need to sift through and maintain. If we need to refer to this in the future we can look at the Git history.

Reply:
I got rid of this file completely.